### PR TITLE
Updated changelog for 0.8.0 release prep

### PR DESCRIPTION
**Pull request by kaoutar55**
## Related issues

N/A

## Description

Release 0.8.0 prep

## Details

Updated the changeling

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,11 +12,13 @@ The format is based on [Keep a Changelog], and this project adheres to
...
 * `Security` in case of vulnerabilities.
 
-## [Unreleased]
+## [0.8.0] - 2023/07/14
 
 
 ### Added
 
+* Added new tutorial notebooks to cover the concepts of training,
+ hardware-aware training, post-training calibration, and extending aihwkit functionality
```
Comment (maljoras) [inline CHANGELOG.md:21]: If you add it you need to link it to the PRs: should be (\#518, \#523, \#526)

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,11 +12,13 @@ The format is based on [Keep a Changelog], and this project adheres to
...
 * Calibration of input ranges for inference (\#512)
 * New analog in-memory training algorithms: Chopped Tiki-taka II (\#512)
 * New analog in-memory training algorithms: AGAD (\#512)
```
Comment (maljoras) [inline CHANGELOG.md:24]: There is the link below (at the end of the file) which needs to be updated as well

**Review by maljoras — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (kaoutar55) [reply CHANGELOG.md:21]: Thanks @maljoras will make the change.

File: CHANGELOG.md (hunk 2 above)
Comment (kaoutar55) [reply CHANGELOG.md:23]: @maljoras I fixed both.

**Review by maljoras — APPROVED** [review]

**Review by kkvtran — APPROVED** [review]

**Review by maljoras — APPROVED** [review]

**Pull request merged**